promotion/datastore.go

```diff
@@ -450,28 +453,34 @@ func (pg *Postgres) GetAvailablePromotions(platform string, legacy bool) ([]Prom
 	statement := `
 		select
 			promotions.*,
+			coalesce(claims.legacy_claimed, false) as legacy_claimed,
```

Comment:
this can also just be `false` - `GetAvailablePromotions` should only include promotions that are globally advertised (as opposed to advertised to particular wallets - which is what re-claims are)